docs/docs/getting-started.server.mdx

```diff
@@ -481,6 +481,42 @@ Which in turn lets us choose whether to use the `@mdx-js/mdx` or
 
 ### Site generators
 
+#### Astro
+
+[Astro](https://astro.build/) has an [official MDX integration](https://docs.astro.build/guides/integrations-guide/mdx/).
+Astro recommends using [the `astro add` CLI](https://docs.astro.build/en/guides/integrations-guide/#automatic-integration-setup)
+to install dependencies and update your config automatically:
+
+```sh
+# Using npm
+npx astro add mdx
+# Using yarn
+yarn astro add mdx
+# Using pnpm
+pnpx astro add mdx
+```
+
+This base setup allows you to import Markdown,
+[Astro components](https://docs.astro.build/en/core-concepts/astro-components/),
+and other MDX files as components.
```

Comment:
```suggestion
The setup generated with `astro add` lets you import Astro components and
other markdown and MDX files as components.
```

Several small edits, particularly:

* ~`This`~ (explain what this is, that it is generated by the previous command)
* ~`base`~ (this word implies that you should likely do more, which is covered later; without the word, it reads to me like it is good enough on its own, and the next point is a great to have)
* `allows` -> `lets` (shorter, just as clear, less jargon-y)
* `Markdown` -> `markdown` (consistent with rest of site, I prefer it because it stresses that markdown is a family of loose formats, rather than standard and specified language)
* reordered to group MDX and markdown together, I feel they make more sense together than when there’s a different format in between that splits them up?
* removed a link, I’m assuming that folks already know about Astro components, or they can find out about them on the Astro site